app/helpers/comments_helper.rb

```diff
@@ -269,6 +279,19 @@ def do_cancel_delete_comment_link(comment)
     end
   end
 
+  def freeze_comment_button(comment)
+    if comment.on_ice?
+      button_to ts("Unfreeze"), unfreeze_comment_path(comment), method: :put
```

Comment:
These buttons won't work without JS, but we can live with that.

Reply:
I think that's only true if it's `link_to`, where we rely on JavaScript to make a link behave like a button. This seems to work fine without JavaScript, since it's a form with a submit button:

```
<li>
  <form class="button_to" method="post" action="/comments/123/unfreeze">
    <input type="hidden" name="_method" value="put">
    <input type="submit" value="Unfreeze">
    <input type="hidden" name="authenticity_token" value="hulk-SMASH">
  </form>
</li>
```